docs/src/api/class-page.md

```diff
@@ -457,10 +457,9 @@ page.
 
 Adds a script which would be evaluated in one of the following scenarios:
 * Whenever the page is navigated.
-* Whenever the child frame is attached or navigated. In this case, the script is evaluated in the context of the newly
-  attached frame.
+* Whenever a child frame is attached or navigated. In this case, the script is evaluated (repeatedly) in the context of every individual child frame.
```

Comment:
`(repeatedly) in the context` turns this doc from being formally correct to being incorrect. Execution context is an established term that describes the JavaScript world in the frame. addInitScript will only run once in each execution context. Note that when there are other worlds (content scripts), it'll run in them as well.